recipe.cake

```diff
@@ -1,8 +1,8 @@
-#load nuget:https://www.myget.org/F/cake-contrib/api/v2?package=Cake.Recipe&version=0.3.0-unstable0427
+#load nuget:https://www.myget.org/F/cake-contrib/api/v2?package=Cake.Recipe&prerelease
```

Comment:
Please pin it to a specific build, I'd like to ensure that the build is repeatable.